main.cpp

```diff
@@ -56,10 +57,29 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    int coins = 5;
+    int pockets = -4;
+    short smallerNumber = 987;
+    short martin = -777;
+    short stop = 29;
+    long bigNumber = -2389898;
+    long realBig = 239838943;
+    long seriouslyLarge = 2000000000;
+    float decimals = 0.5f;
+    float rootBeer = 123.48f;
+    float hover = 83838.355f;
+    double whiskey = 989898.34f;
+    double dutch = 23229939.1114f;
+    double dip = 71238.1487f;
```

Comment:
Initialize doubles with doubles.  You have written `float` numbers on the right side of the `=`
- [ ] revise